src/status_im/network/events.cljs

```diff
@@ -0,0 +1,58 @@
+(ns status-im.network.events
+  (:require [status-im.network.core :as network]
+            [status-im.utils.handlers :as handlers]
+            [re-frame.core :as re-frame]
+            [status-im.utils.utils :as utils]))
+
+(handlers/register-handler-fx
```

Comment:
with new fx/defn we don't need to separate events anymore, just use {:events []} in fx/defn and you can move all these handlers to core

Reply:
The whole objective was rather to rearrange code so that it can be compiled as a module, not to rewrite it. I know about this but it can be rewritten later if necessary.